### PR TITLE
Readme and Preheat titles color

### LINKED ISSUE #1232: [BUG] Preheat titles color
### Steps to reproduce

1. Change background color to white.
2. Preheat menu titles are also white.

### Hardware Variant
TFT35 v3
### TFT Firmware Version & Main Board Firmware details
CONFIG_VERSION 20200810

### Additional Information

In preheat.c I changed the title color to the code below. I'm not great with github commits so I was hoping someone could add this. Thank you.

  ```
//set preheat title properties
  lvIcon.lines[0].h_align = CENTER;
  lvIcon.lines[0].v_align = TOP;

  //GTL changed to match font color
  GUI_SetColor(infoSettings.font_color);
  //lvIcon.lines[0].fn_color = LCD_WHITE;

  lvIcon.lines[0].text_mode = GUI_TEXTMODE_TRANS;
  lvIcon.lines[0].pos = preheat_title;
  lvIcon.lines[0].large_font = false;
```

### PR DESCRIPTION
**Pull request by oldman4U**
Added information about Reset default settings ... to the ReadMe

Changed Preheat titles color according to gordo3di

resolves #1232

### CODE REVIEW

**Comment by oldman4U** [comment]
You saw it better, even your eyes are two knobs;-)

Should be fixed now, thank you

**Comment by oldman4U** [comment]
Converted to Draft because the reset after the fw update makes no sense.

**Comment by GregSKR** [comment]
"element4: **Ne** or several language file(s) (optionally)"

Maybe you wanted to write "One"? 😉

File: TFT/src/User/Menu/PreheatMenu.c [hunk 1]
```diff
@@ -28,10 +28,14 @@ void refreshPreheatIcon(int8_t preheatnum, int8_t icon_index, const ITEM * menui
...
-  //set preheat title properties
+//set preheat title properties
   lvIcon.lines[0].h_align = CENTER;
   lvIcon.lines[0].v_align = TOP;
-  lvIcon.lines[0].fn_color = LCD_WHITE;
+
+//GTL changed to match font color
+  GUI_SetColor(infoSettings.font_color);
+//lvIcon.lines[0].fn_color = LCD_WHITE;
+
```
Comment (guruathwal) [inline TFT/src/User/Menu/PreheatMenu.c:38]: Instead, use this. it should work after #1384 
```suggestion

//GTL changed to match font color
  lvIcon.lines[0].fn_color = infoSettings.font_color;
  //lvIcon.lines[0].fn_color = LCD_WHITE;
```

**Comment by oldman4U** [comment]
> "element4: **Ne** or several language file(s) (optionally)"
> 
> Maybe you wanted to write "One"? 😉

Thank you GregSKR. Found some additional typos and another broken link...

File: TFT/src/User/Menu/PreheatMenu.c (hunk 1 above)
Comment (oldman4U) [reply TFT/src/User/Menu/PreheatMenu.c:38]: OK, thank you. Changed it and will push it asap.

**Comment by oldman4U** [comment]
OK. Currently there is no information about the reset in the Readme. But @kisslorand and @mehmetsutas figured out, that under certain circumstances this is required. So I added the information, but for me the order I defined makes no sense:

**step1:** Copy your self compiled firmware or the BIGTREE_TFT*_V*.*.*.bin, plus the TFT*` folder and the config.ini to the root of a blank SD card that is <8GB and formatted as FAT32:

**step2:** Place SD card with the `BIGTREE_TFT*_V*.*.*.bin`, the`TFT*` folder and the config.ini into the TFT's SD card reader and reset your TFT (or optionally - power cycle your printer) to start the update process.

**step3:** Remove the SD card from the TFT and restart the printer.

**step4:** On the TFT click on Menu - Settings - Feature and navigate to the last page. Click on "Reset default settings ..." to refresh the content of the TFT's memory.

**step5:** Restart the printer to finish the installation

Shouldn't the reset be number 1 or 2, so before the firmware and therefore also the config.ini update? So

#1 Start printer, reset default settings and turn printer OFF
#2 Copy stuff on TFT
#3 insert sd card and start printer
#4 after successful update remove sd card
#5 restart printer

Or what would be the right order?

Please let me know

**Comment by kisslorand** [comment]
Actually I made a reset by having the "reset.txt" file on the SD Card together with "config.ini". I don't know what was the order the TFT did things.

**Comment by mehmetsutas** [comment]
@oldman4U that order should work fine.

**Comment by oldman4U** [comment]
The order with the reset at the beginning or the end?

mehmetsutas <notifications@github.com> schrieb am So. 13. Dez. 2020 um
20:16:

**Comment by oldman4U** [comment]
Wouldn‘t it be better to have the reset as part of the firmware update?

kisslorand <notifications@github.com> schrieb am So. 13. Dez. 2020 um 19:50:

**Comment by oldman4U** [comment]
kisslorand. I thought the reset.txt only forces the TFT to show the calibration again. Can you see in the code what else it does?

**Comment by kisslorand** [comment]
I cannot see what it does, it is in the bootloader. I am not good at disassembling.
If "Reset.txt" might resets the TFT calibration, than my config got reset by itself because I didn't press "Reset default settings", that's for sure.

**Comment by guruathwal** [comment]
@oldman4U  It resets every setting to default on the TFT firmware.

**Comment by oldman4U** [comment]
Hi Gurmeet.

Thank you

In case we add the recommendation to reset the TFT using the "Reset default settings" function during a firmware update, what would be the best time to do it. Before the update is done?

**Comment by mehmetsutas** [comment]
@oldman4U It should be dome just after flashing the firmware. 

 Before the update there is no change on config structure, so resetting would be useles if done before update.

**Comment by guruathwal** [comment]
@oldman4U The settings are reset automatically during boot whenever it detects outdated setting signatures and asks the user to update the required files.
This screen is displayed:
image

**Comment by oldman4U** [comment]
But then there is a bug or the user has to upload the config.ini twice
because after the reset the information from the config.ini is reset to
something else.

mehmetsutas <notifications@github.com> schrieb am Mo. 14. Dez. 2020 um
12:29:

**Comment by oldman4U** [comment]
> @oldman4U The settings are reset automatically during boot whenever it detects outdated setting signatures and asks the user to update the required files.
> This screen is displayed:
> image

OK, so an additional reset is not needed and therefore I will remove this information from the ReadMe again.

;-)

**Comment by kisslorand** [comment]
> @oldman4U The settings are reset automatically during boot whenever it detects outdated setting signatures and asks the user to update the required files.

The last days I played with firmwares post and prior PR #1333. I was never asked to upload new config.ini, I always had to manually reset either by "reset.txt" or inside the menu. The TFT was always messed up without a reset .

**Comment by oldman4U** [comment]
On Dec. 11 (with #1333) we got a new feature in config.ini called "Enable print summary popup".

So I took a firmware from before (Dec. 10th for example) verified that the config.ini does not include "Enable print summary popup" an updated my printer with firmware, config.ini and TFT folder.
After that I updated my printer ONLY with the FIRMWARE from Dec. 11. NO config.ini and NO TFT folder but made sure, that it has the new entry in config.ini.
The update was successful but WITHOUT the warning that the config.ini or any other element is wrong.

Possible there is a bug in this area??

**Comment by guruathwal** [comment]
@oldman4U This has happened many times in the past also because no one wants to read.
image

**Comment by oldman4U** [comment]
OK, so there is really a problem because the version has not been updated. I guess this happened to @mehmetsutas so I add him to this ticket in case he wants to contribute in the future.

I remove the rest information.

Thank you

**Comment by mehmetsutas** [comment]
Excuse me for the inconvinience. I think this is solved now. I remember investigating those 2 lines, but I think I have interpreted the do not modify warning for both of the definitions and skipped the PARA_SIGN definition.

**Pull request merged**